tools/accuracy_checker/accuracy_checker/data_readers/data_reader.py

```diff
@@ -14,16 +14,17 @@
 limitations under the License.
 """
 
-from pathlib import Path
-from functools import singledispatch
-from collections import OrderedDict, namedtuple
 import re
 import wave
+from collections import OrderedDict, namedtuple
+from functools import singledispatch
+from pathlib import Path
 
 import cv2
-from PIL import Image
+import lmdb
```

Comment:
please wrap to Unsupported Package import lmdb

Reply:
Wrapped.